modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator-to-minor-version_{context}"]
+= Upgrading the MetalLB Operator
+
+Manually upgrade the MetalLB Operator to the next minor version by using the {product-title} web console.
```

Comment:
I am still not 100% sure about this. 
I'd try to make a distinction between minor upgrades (4.11.0 to 4.11.z) and upgrades coming from a version < 4.11